disnake/ext/commands/bot.py

```diff
@@ -252,6 +258,11 @@ class InteractionBot(InteractionBotBase, disnake.Client):
         Only applicable if the ``localization_provider`` parameter is not provided.
         Defaults to ``False``.
 
+        .. versionadded:: 2.5
+    i18n: :class:`.LocalizationProtocol`
```

Comment:
missing a new line to be consistent
```suggestion
        .. versionadded:: 2.5

    i18n: :class:`.LocalizationProtocol`
```